code/datums/ai_laws.dm

```diff
@@ -360,7 +360,7 @@
 	while (supplied.len < number + 1)
 		supplied += ""
 
-	supplied[number + 1] = law
+	supplied[number + 1] = replace_html_codes(law)
```

Comment:
most of AI laws are done by player input and I believe there would be some rare case that is input by an unregular way besides AI module. This will cover that if case.